# Hold data extracted from Sense HAT environmental sensors (humidity and pressure)
Header header
# Percentage of relative humidity 
float64 humidity
# Temperature in degrees Celsius from the humidity sensor
float64 temperature_from_humidity
# Temperature in degrees Celsius from the pressure sensor
float64 temperature_from_pressure
# Pressure in Millibars
float64 pressure
